awx/ui_next/src/components/MultiSelect/MultiSelect.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import MultiSelect, { _MultiSelect } from './MultiSelect';
+import { mountWithContexts } from '../../../testUtils/enzymeHelpers';
+
+describe('<MultiSelect />', () => {
+  const associatedItems = [
+    { name: 'Foo', id: 1, organization: 1 },
+    { name: 'Bar', id: 2, organization: 1 },
+  ];
+  const options = [{ name: 'Angry', id: 3 }, { name: 'Potato', id: 4 }];
+
+  test('Initially render successfully', () => {
+    const renderChips = jest.spyOn(_MultiSelect.prototype, 'renderChips');
+    const wrapper = mountWithContexts(
+      <MultiSelect
+        onAddNewItem={jest.fn()}
+        onRemoveItem={jest.fn()}
+        associatedItems={associatedItems}
+        options={options}
+      />
+    );
+    const component = wrapper.find('MultiSelect');
+
+    expect(renderChips).toBeCalled();
+    expect(component.state().chipItems.length).toBe(2);
+  });
+  test('handleSelection add item to chipItems', async () => {
+    const wrapper = mountWithContexts(
+      <MultiSelect
+        onAddNewItem={jest.fn()}
+        onRemoveItem={jest.fn()}
+        associatedItems={associatedItems}
+        options={options}
+      />
+    );
+    const event = { preventDefault: () => {} };
+    const component = wrapper.find('MultiSelect');
+    component.instance().handleSelection(event, { name: 'Apollo', id: 5 });
```

Comment:
can you simulate the click event rather than accessing the instance method directly?